packages/runtime/src/state/react-page.ts

```diff
@@ -118,71 +131,26 @@ export function getPropControllers(state: State, documentKey: string, elementKey
   )
 }
 
-function normalizeElement(
-  element: Documents.Element,
-  descriptors: Map<string, Record<string, PropControllers.PropControllerDescriptor>>,
-): Map<string, Documents.Element> {
```

Comment:
Somewhat misleading name, this is the on-demand element tree flattening that was done every time we called `getDocumentElements`

Reply:
Transposed into `buildElementTree` [here](https://github.com/makeswift/makeswift/pull/894/files#diff-4c77c71de0cb4022cef56ea1671d57f758f297a229dab9ae8caf6f307c7b7918R124).